memory_addr/Cargo.toml

```diff
@@ -0,0 +1,15 @@
+[package]
+name = "memory_addr"
+version = "0.2.1"
```

Comment:
Use workspace version and make both crates the same version?

Reply:
OK

Reply:
Updated